source/reference/method/MongoDBDatabase-listCollectionNames.txt

```diff
@@ -74,8 +74,8 @@ Parameters
 Return Values
 -------------
 
-An :php:`Iterator <class.iterator.php>`, which provides the name of each
-collection in the database.
+An :php:`Iterator <manual/en/class.iterator.php>`, which provides the
+name of each collection in the database.
```

Comment:
I like this wording more than "contains" or "stored in". I expect this was just written at a different time than the other pages, which is why they're inconsistent.

Saying that an "iterator provides something" is a concise way to convey the technical aspect of it yielding results when traversed (in simpler terms).